Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM jekyll/jekyll
```

Comment:
do you know if this is always aligned with the jekyll image that runs on github? that would be cool

Reply:
Ah, I didn't think about that. Nope!
[Github page dependency versions](https://pages.github.com/versions/) mentions that they are using 3.8.5
My dockerfile uses the latest image.